src/boxed.rs

```diff
@@ -115,3 +115,14 @@ where
 }
 
 impl<P, Item> PredicateBoxExt<Item> for P where P: Predicate<Item> {}
+
+#[cfg(test)]
+mod test {
+    use crate::prelude::*;
+
+    #[test]
+    fn unsized_boxed() {
```

Comment:
Does this commit compile?

Reply:
No, because I had misunderstood the "Add tests in a commit before their feature or fix, showing the current behavior" part of CONTRIBUTING.md (I thought you wanted a demonstration of the current behavior of "it does not compile", and figured you'd squash the commits after confirming). Looking at the provided example more closely, I now see the `unwrap_err()` to `unwrap()` change to the tests in the fix there.

I do not know how to do the equivalent here, since the problem is at compile time (I do not know how to easily write a test that asserts that something does not compile). Should I just squash the commits, or can you point me in the right direction?

Reply:
There isn't; you just add the test in the current commit.

"Encourage" was doing a lot of heavy lifting in that text.  I've tried to clarify things in the template repo (https://github.com/epage/_rust/commit/e783559994c5e56d4d3327443dd70abb0f95ef68) which will get merged here next time I do updates.

Reply:
That was on me, I should've looked at the example commits more closely (but thanks for those docs tweaks, the new wording would have made me understand without having to look at the commits).

I've squashed the commits.